src/Components/ProjectForm.js

```diff
@@ -0,0 +1,162 @@
+import React from 'react';
+import { ReactSession } from "react-client-session";
+import styled from "styled-components";
+import TextField from '@mui/material/TextField';
+import Service from "../Service";
+
+const Styles = styled.div`
+ background: #218888;
+ padding: 20px;
+
+ h3 {
+   border-bottom: 1px solid white;
+   color: #6f6f6f;
+   font-family: sans-serif;
+   font-size: 20px;
+   font-weight: 600;
+   line-height: 24px;
+   padding: 10px;
+   text-align: center;
+ }
+
+ form {
+   background: white;
+   border: 1px solid #dedede;
+   display: flex;
+   flex-direction: column;
+   justify-content: space-around;
+   margin: 0 auto;
+   max-width: 1000px;
+   padding: 50px 200px;
+ }
+
+ input {
+   border: 1px solid #d9d9d9;
+   border-radius: 4px;
+   box-sizing: border-box;
+   padding: 10px;
+   width: 100%;
+ }
+
+ label {
+   color: #6f6f6f;
+   display: block;
+   font-family: sans-serif;
+   font-size: 14px;
+   font-weight: 500;
+   margin-bottom: 5px;
+ }
+
+ .error {
+   color: red;
+   font-family: sans-serif;
+   font-size: 12px;
+   height: 30px;
+ }
+
+ .submitButton {
+   background-color: #6f6f6f;
+   color: white;
+   font-family: sans-serif;
+   font-size: 14px;
+   margin: 20px 0px;
+`;
+
+function ProjectForm() {
+  const [name, setName] = React.useState("");
+  const [description, setDescription] = React.useState("");
+  const [imageURL, setImageURL] = React.useState("");
+  const [message, setMessage] = React.useState("");
+  const [user, setUser] = React.useState([""]);
+
+  React.useEffect(() => {
+    setUser(ReactSession.get("username"));
+  }, []);
+
+  const handleNameChange = (e) => {
+    setName(e.target.value);
+  }
+
+  const handleDescriptionChange = (e) => {
+    setDescription(e.target.value);
+  }
+
+  const handleImageURLChange = (e) => {
+    setImageURL(e.target.value);
+  }
+
+  const handleSubmit = event => {
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/CSC510-Group-25/feature-hunt/pullRequest?prid=8358184)
```suggestion
  const handleSubmit = (event) => {
```